acq4/util/imaging/frame_display.py

```diff
@@ -8,6 +8,10 @@
 from .contrast_ctrl import ContrastCtrl
 
 
+MAX_FRAMES_PER_SECOND = 30
+S_PER_FRAME = 1.0 / MAX_FRAMES_PER_SECOND
+MS_PER_FRAME = int(S_PER_FRAME * 1000)
```

Comment:
I don't love that these two vars have similar names but different data types.

Reply:
Why global variables rather than instance variables?